tests/openstack/task.yaml

```diff
@@ -0,0 +1,101 @@
+summary: Test the openstack backend.
+
+systems: [ubuntu-22.04-64-devstack]
+
+kill-timeout: 30m
+
+prepare: |
+    if [ ! -f .spread-reuse.yaml ]; then
+        touch /run/spread-reuse.yaml
+        ln -s /run/spread-reuse.yaml .spread-reuse.yaml
+    fi
+
+    sudo -u stack -i /opt/stack/devstack/unstack.sh
+    sudo -u stack -i /opt/stack/devstack/stack.sh
+
+execute: |
+    # Check the instances which cannot be accessed are discarded
+    # The instance was created and the status has to be active to
+    # fail trying to access through ssh
+    spread openstack:cirros-64: -v -reuse -resend &> task.out || true
+    grep 'cannot find ready marker in console output for .*: timeout reached' task.out
```

Comment:
This is not a great method to verify the contents, because on failure we'll get nothing, and won't know why we got nothing.  I believe we have common practices for this in the Spread world.  How do they look like?